qa/tasks/tests/requirements.txt

```diff
@@ -0,0 +1,6 @@
+boto
+gevent
+httplib2
+pytest
+python-dateutil
+teuthology
```

Comment:
Small nit: Looks like GitHub wants a newline at the end of the file (see the "Files changed" tab where it shows a red circle with a line through it). I think this is just GitHub's convention.